### DISCUSSION TITLE
Make Grafana compatible with `amtool`

### DISCUSSION THREAD

**Discussion by gotjosh** [grafana/grafana#37649] [Alerting]
Grafana supports all of the operations via the API that the upstream Alertmanager does. We should attempt (via testing and changing accordingly) that it's 100% compatible with `amtool`.

This includes supporting version checks which are not currently possible: https://github.com/prometheus/alertmanager/pull/2672

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
